src/qml/pages/onboarding/OnboardingCover.qml

```diff
@@ -18,19 +18,19 @@ Page {
         interactive: false
         orientation: Qt.Horizontal
         InformationPage {
-            navRightDetail: NavButton {
-                iconSource: "image://images/info"
-                iconHeight: 24
-                iconWidth: 24
-                iconColor: Theme.color.neutral0
-                iconBackground: Rectangle {
-                    radius: 12
-                    color: Theme.color.neutral9
-                }
-                onClicked: {
-                    introductions.incrementCurrentIndex()
-                }
-            }
+            // navRightDetail: NavButton {
```

Comment:
how crucial is to have the info on the cover page?

The challenge here is that we don't fully initialize nodeModel and optionsModel, until we finish the Onboarding process, and they are referenced in AboutOptions.qml and DeveloperOptions.qml. 

If it's important I will look into a way to accomodate for that, also it could be a follow up PR. What are your thoughts?

Reply:
I think the about page is important but we might need to just disable the options portion until onboarding is completed

Reply:
sounds good... do you think disabling the options portion should be part of this PR?